lib/components/atoms/Icon/Icon.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+/**
+ *
+ * Icon
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+import classnames from 'classnames';
+
+import styles from './Icon.style';
+import type { Props } from './types';
+
+const Icon = ({ className, title, ...others }: Props): Node => {
+  return (
+    <div role="presentation" className={classnames('icon-wrapper', className)} {...others}>
```

Comment:
Can you change this icon to span, so that it's inline. Because its a DIV you will not be able to place icon in front of a DIV